src/components/PermissionsManagement/UserPermissionsPopUp.jsx

```diff
@@ -94,6 +135,7 @@ function UserPermissionsPopUp({
             autoClose: 10000,
           });
           setToastShown(true);
+          logPermissionChanges(logPermissionChangesUrl, permissionChangeLog);
```

Comment:
```suggestion
```
This is not atomic. This request could fail after the permission change request succeeds, so it wouldn't show up in the history. If a dev wanted to test updating permissions manually with something like Postman, they could avoid also logging  those changes which could cause confusion or potential problems.
Logging permission changes should be part of the same API call as updating the user's permissions.